File: Java/CodingBat/roundSum.java
// http://codingbat.com/prob/p186753

public int roundSum(int a, int b, int c) {
  return round10(a) + round10(b) + round10(c);
}

public int round10(int num) {
  int rightMostDigit = num % 10;
  return (rightMostDigit >= 5) ? (num + (10 - rightMostDigit)) : (num - rightMostDigit);
}

File: Python/CodingBat/round_sum.py
# http://codingbat.com/prob/p179960

def round_sum(a, b, c):
  sum = round10(a) + round10(b) + round10(c)
  
  return sum
  
def round10(num):
  last_digit = num % 10
  
  if last_digit >= 5:
    return num + (10 - last_digit)
  else:
    return num - last_digit
